src/Config.cpp

```diff
@@ -75,6 +75,19 @@ Config::Config() {
         char value = std::any_cast<char>(elem);
         return value;
     };
+    auto validate_quote = [&](std::string key, std::any elem) {
+        bool value = std::any_cast<bool>(elem);
+        if (key == "double_quote_to_single_quote") {
+            if (value && (node_["single_quote_to_double_quote"]).as<bool>()) {
+                throw logic_error("[ERROR] Configuration value is conflicting with other value");
```

Comment:
You know the values that are conflicting so you should report them.
This will aid the user to fix the issue.